<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QClickableLabel" name="zoomedImageLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>480</width>
       <height>320</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>480</width>
       <height>320</height>
      </size>
     </property>
     <property name="text">
      <string>Image</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QClickableLabel" name="zoomedDepthMapLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>480</width>
       <height>320</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>480</width>
       <height>320</height>
      </size>
     </property>
     <property name="text">
      <string>Image</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QClickableLabel" name="reconstructionLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>480</width>
       <height>256</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>480</width>
       <height>256</height>
      </size>
     </property>
     <property name="text">
      <string>Image</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QClickableLabel</class>
   <extends>QLabel</extends>
   <header>qclickablelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
